src/main/java/de/tum/in/www1/artemis/web/rest/GradingScaleResource.java

```diff
@@ -90,6 +96,20 @@ public ResponseEntity<GradingScale> getGradingScaleForExam(@PathVariable Long co
         return gradingScale.map(ResponseEntity::ok).orElseGet(() -> ResponseEntity.ok(null));
     }
 
+    /**
+     * Search for all modeling exercises by title and course title. The result is pageable since there might be hundreds
```

Comment:
wrong comment, probably a copy paste error

Reply:
Copy paste error is replaced by actual comment